CMakeLists.txt

```diff
@@ -8,6 +8,7 @@ add_subdirectory(extern/pybind11)
 if(MSVC)
```

Comment:
```cmake
set(CMAKE_CXX_STANDARD 17)

if(MSVC)
    add_compile_options("/W4" "/Ox" "/fp:fast")
endif()
if(UNIX)
    add_compile_options("-Ofast")
endif()
```

I think we can simplify here. Could you try this out?

Reply:
Something is off, why is only codacy static code analysis running? 🤔

Reply:
The tests are also running in the workflows as far as I can see here https://github.com/danieleboaretti/affine_transform_nd/actions (the publish workflows fail due to missing rights and that makes sense to me 😄 )

The tests pass with the simplification you suggested! 🙌

Reply:
I see, so they are triggered in your fork not this main repository. Kind of difficult to keep track of. I will have to see if I can play around with the settings. Thanks for the hint.